### PR TITLE
SVE Fixes #6 (or 7, or whatever)

### LINKED ISSUE #6: FileNotFoundException due to file case mismatch
I installed the 4.5.3 release to play with the new settings, but it mostly failed to load (the starting season was changed, but nothing else appeared to have loaded). After launching from a terminal to see logs, I saw 3 errors, one of which mentioned that `Stardew Valley/Content/Data/Hats.xnb` could not be found. In that directory, there is instead a `hats.xnb` (and a bunch of versions for other languages, all lowercase), and most files in the directory start with uppercase names. I use linux, and have a case sensitive filesystem. Creating a symlink with the appropriately capitalized name caused it to launch without issue, and it appears to load correctly.

I haven't specifically noticed errors for other things with lowercase data files in that directory, but it could be worth checking the case the code is expecting for those too, just in case.

### PR DESCRIPTION
**Pull request by Witchybun**
- Fixes Apples warp if you decide to return the vineyard to normal.
- Pass at fixing the shop; during transition there was an error on where the shop was meant to be in the first place.
- Update mod version for CP mods (note that its been updated)

### CODE REVIEW

File: StardewArchipelago/Constants/ModNames.cs [hunk 1]
```diff
@@ -27,8 +27,8 @@ public class ModNames
         public const string ALECTO = "Alecto the Witch";
         public const string DISTANT_LANDS = "Distant Lands - Witch Swamp Overhaul";
 
-        public const string AP_SVE = "StardewArchipelago - Stardew Valley Expanded";
-        public const string AP_ALECTO = "StardewArchipelago - Alecto the Witch";
-        public const string AP_JASPER = "StardewArchipelago - Professor Jasper Thomas";
+        public const string AP_SVE = "StardewAP - SVE Patch";
+        public const string AP_ALECTO = "StardewAP - Alecto the Witch Patch";
```
Comment (agilbert1412) [inline StardewArchipelago/Constants/ModNames.cs:31]: why not "Alecto Patch" and "Jasper Patch", to simplify the same as much as possible and reduce similarity with the original while maintaining clarity?

File: StardewArchipelago/GameModifications/EntranceRandomizer/EquivalentWarps.cs [hunk 2]
```diff
@@ -207,5 +235,51 @@ private bool IsGrandpaShed(string area, out string correctArea)
             correctArea = area;
             return false;
         }
+
+        private bool IsAuroraVineyard(string area, out string correctArea)
+        {
+            foreach (var auroraVineyardLocations in _auroraVineyardLocations)
```
Comment (agilbert1412) [inline StardewArchipelago/GameModifications/EntranceRandomizer/EquivalentWarps.cs:241]: Location should be singular

File: StardewArchipelago/GameModifications/EntranceRandomizer/EquivalentWarps.cs [hunk 3]
```diff
@@ -207,5 +235,51 @@ private bool IsGrandpaShed(string area, out string correctArea)
...
+                return true;
+            }
+
+            correctArea = area;
+            return false;
+        }
+
+        private bool IsAuroraVineyardCellar(string area, out string correctArea)
+        {
+            foreach (var auroraVineyardCellarLocations in _auroraVineyardCellarLocations)
```
Comment (agilbert1412) [inline StardewArchipelago/GameModifications/EntranceRandomizer/EquivalentWarps.cs:264]: Location should be singular

**Review by agilbert1412 — APPROVED** [review]

File: StardewArchipelago/Constants/ModNames.cs (hunk 1 above)
Comment (Witchybun) [reply StardewArchipelago/Constants/ModNames.cs:31]: Because then its too generic with any other patch.  It should say what the patch is for.

Comment (agilbert1412) [reply StardewArchipelago/Constants/ModNames.cs:31]: Possibly I didn't express this right.
I meant replace "Alecto the Witch Patch" with "Alecto Patch". So the full mod name would go from "StardewAP - Alecto the Witch Patch" to "StardewAP - Alecto Patch", which I think would be just as specific as the current name, but less wordy.

I did not mean to get rid of the "StardewAP - " part

Comment (Witchybun) [reply StardewArchipelago/Constants/ModNames.cs:31]: A bit tedious to do again, but alright.

File: Documentation/Supported Mods.md [hunk 4]
```diff
@@ -14,7 +14,7 @@ A reasonable effort will be made to stay up to date, but it has to be understood
 The following mods are currently supported:
 
 General: 
-- [Stardew Valley Expanded](https://www.nexusmods.com/stardewvalley/mods/3753), by FlashShifter, Version 1.14.24
+- [Stardew Valley Expanded](https://www.nexusmods.com/stardewvalley/mods/3753), by FlashShifter, Version 1.14.24 (including additional patch)
```
Comment (agilbert1412) [inline Documentation/Supported Mods.md:17]: Could you link to your own releases page here? You could make the text "Additional patch" into a link to it in the 3 instances

**Review by agilbert1412 — APPROVED** [review]

**Review by agilbert1412 — APPROVED** [review]

**Pull request merged**